packages/main/src/Panel.js

```diff
@@ -139,23 +145,40 @@ const metadata = {
 			type: String,
 			defaultValue: "",
 		},
+
+		/**
+		 * Receives id(or many ids) of the elements that label the header of the panel
+		 *
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.0-rc.9
+		 */
+		headerAriaLabelledBy: {
```

Comment:
Should be headerAriaLabelledby, so the users can write header-aria-labelledby (not header-aria-labelled-by as now)

Reply:
Done